lms/djangoapps/discussion_api/views.py

```diff
@@ -168,6 +180,13 @@ class docstring.
         """
         return Response(create_thread(request, request.DATA))
 
+    def partial_update(self, request, thread_id):
```

Comment:
In the doc it is called topic_id, in this function it is called thread_id. Should we make a note somewhere that these are equivalent (unless I'm mistaken)